examples/demo-cluster.php

```diff
@@ -60,6 +60,7 @@
         true
     );
 
+    /** @psalm-suppress ForbiddenCode */
```

Comment:
I dislike having such annotations in the code. I suggest suppressing this rule for the examples directory in psalm configuration instead.

Reply:
ok